integration-tests/pipelines/e2e-tests-pipeline.yaml

```diff
@@ -12,36 +12,54 @@ spec:
       default: "build-templates-e2e"
       description: "namespace where e2e-tests will be executed"
   tasks:
+    - name: test-metadata
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/tekton-integration-catalog.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: common/tasks/test-metadata/0.1/test-metadata.yaml
     - name: prepare-e2e-tests
       params:
         - name: SNAPSHOT
           value: $(params.SNAPSHOT)
+      when:
+        - input: "$(tasks.test-metadata.results.test-event-type)"
+          operator: in
+          values: ["pull_request"]
       taskSpec:
         params:
         - name: SNAPSHOT
         results:
-          - name: custom-source-build-bundle
+          - name: custom-build-bundle
             description: "custom bundle for source build pipeline"
         steps:
           - name: e2e-tests
-            image: quay.io/konflux-ci/appstudio-utils:ab6b0b8e40e440158e7288c73aff1cf83a2cc8a9@sha256:24179f0efd06c65d16868c2d7eb82573cce8e43533de6cea14fec3b7446e0b14
+            image: quay.io/konflux-ci/appstudio-utils:48c311af02858e2422d6229600e9959e496ddef1@sha256:91ddd999271f65d8ec8487b10f3dd378f81aa894e11b9af4d10639fd52bba7e8
             env:
             - name: SNAPSHOT
               value: $(params.SNAPSHOT)
             script: |
+              set -e
```

Comment:
Do we really need it?

Reply:
does not wanted to continue after we get the first failure in the task, so added it. Does it do any harm in keeping it?